app-config.yaml

```diff
@@ -19,6 +19,9 @@ app:
         links:
           - url: https://discord.gg/MUpMjP2
             title: '#backstage'
+  featureFlags:
```

Comment:
I think we should remove this as it doesn't really give any benefit to having this. I am also keen to have something like this here but commented out. If you want to do that lets give it a more meaningful name as well. 

```yaml
- name: tech-radar
  description: Enables the tech radar plugin
```